agasc/supplement/magnitudes/mag_estimate.py

```diff
@@ -722,18 +722,19 @@ def calc_obs_stats(telem):
 
     kalman = (telem['AOACASEQ'] == 'KALM') & (telem['AOPCADMD'] == 'NPNT')
     track = (telem['AOACIIR'] == 'OK') & (telem['AOACFCT'] == 'TRAK')
```

Comment:
I have forgotten our process so maybe this should have a comment about why saturated pixel is not included in the filter?

Reply:
It would be a good thing to remember before we try to get a Gaia-based catalog though.

Reply:
I wrote a global comment responding to this.